fabtests/prov/efa/Makefile.include

```diff
@@ -65,4 +68,11 @@ prov_efa_src_fi_efa_exhaust_mr_reg_rdm_pingpong_SOURCES = \
 	$(efa_exhaust_mr_reg_srcs) \
 	$(benchmarks_srcs)
 prov_efa_src_fi_efa_exhaust_mr_reg_rdm_pingpong_LDADD = libfabtests.la
+
+if HAVE_EFADV
+prov_efa_src_fi_efa_rdma_checker_SOURCES = \
+	prov/efa/src/efa_rdma_checker.c
+prov_efa_src_fi_efa_rdma_checker_LDADD = libfabtests.la
```

Comment:
It's dubious to me whether `LDADD` is needed.

Reading https://www.gnu.org/software/automake/manual/html_node/Linking.html I think we can do `prov_efa_src_fi_efa_rdma_checker_LDFLAGS = -lefa` and get rid of `configure.m4`

Reply:
Good call out, It works for me. I added LDFLAGS and dropped https://github.com/ofiwg/libfabric/commit/69011622ba076d73d43352698417f127bb26a3a3.